duet/views/funnel_ios_new.view.lkml

```diff
@@ -0,0 +1,429 @@
+                           , COUNT(DISTINCT CASE WHEN DATE_DIFF(submission_date, first_run_date, day) >= 1 AND DATE_DIFF(submission_date, first_run_date, day) <= 28 AND days_since_seen = 0 THEN submission_date END) AS first_28_days_active_days
+            FROM `mozdata.firefox_ios.baseline_clients_last_seen`
+            WHERE
+              submission_date >= '2022-01-01'
+                      AND first_seen_date >= '2022-01-01'
+              AND NOT (app_display_version = '107.2' AND submission_date >= '2023-02-01') -- incident filter
+            GROUP BY 1, 2, 3
+                    ), ret_aggs as (
+                      SELECT first_seen_date AS date,
+                        country, count(distinct case when first_28_days_active_days > 1 then client_id end) multi_days_users
+                               , count(distinct case when week_4_retention = True then client_id end) week_4_retained
+                      FROM retention
+                      group by 1, 2
+                    )
+                    SELECT date, country, views, downloads, new_profiles, activations, multi_days_users, week_4_retained
+                    FROM top
+                    JOIN bottom USING (date, country)
+                    JOIN ret_aggs USING (date, country)
+;;
+    }
+
+    dimension: country {
+      type: string
+      sql: ${TABLE}.country ;;
+    }
+
+    dimension_group: date {
+      type: time
+      datatype: date
+      timeframes: [date, week, month, year]
+      sql: ${TABLE}.date ;;
+      convert_tz: no
+    }
+
+    measure: views {
+      description: "Unique daily impressions, counted when a customer views the app on the Today, Games, Apps, or Search tabs on the App Store, or on the product page"
+      type: sum
+      sql: ${TABLE}.views ;;
+    }
+
+    measure: downloads {
+      description: "Total downloads, including first-time downloads and redownloads"
+      type: sum
+      sql: ${TABLE}.downloads ;;
+    }
+
+    measure: new_profiles {
+      description: "Unique Client IDs, usually generated when the app is opened for the first time"
+      type: sum
+      sql: ${TABLE}.new_profiles ;;
+    }
+
+    measure: activations {
+      description: "Early indicator for LTV based on days of app open and searches on the first week"
+      type: sum
+      sql: ${TABLE}.activations ;;
+    }
+
+  measure: multi_days_users {
+    description: "Early indicator for LTV based on days of app open in first 28 days"
+    type: sum
+    sql: ${TABLE}.multi_days_users ;;
+  }
+
+  measure: week_4_retained {
+    description: "Early indicator for LTV based on app open in week 4"
+    type: sum
+    sql: ${TABLE}.week_4_retained ;;
+  }
+
+    measure: view_ctr {
+      label: "View Click Through Rate"
+      type: number
+      value_format_name: percent_2
+      sql: ${new_profiles}/ NULLIF(${views},0) ;;
+    }
+
+  measure: download_2_new_profiles {
```

Comment:
I thought we decided to leave this metric out since the top of the funnel with App Store info should be somewhat separate from Telemetry numbers, in particular to keep it consistent with Android, where it doesn't look like a funnel between those steps.

Reply:
I do not recall, but I still think we should have it in the view and not expose it through the dashboard if anything. It can be useful down the line if we have to do a deep dive into the nuances in the funnel later